pkg/storage/postgresql/testdbutils.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2022 Adevinta
+*/
+
+package postgresql
+
+import (
+	"database/sql"
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/go-testfixtures/testfixtures/v3"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/gommon/log"
+)
+
+const (
+	// TestDBName is the testing database name.
+	TestDBName = "vultrackerdb_test"
+	// TestDBUser is the testing database user.
+	TestDBUser = "vultrackerdb_test"
+	// TestDBPassword is the testing database password.
+	TestDBPassword = "vultrackerdb_test"
+	// DBDialect is the testing database dialect.
+	DBDialect = "postgres"
+)
+
+type mockLogger struct {
+	echo.Logger
+}
+
+func (l *mockLogger) Level() log.Lvl {
+	return log.OFF
+}
+
+// FromConnStrToDsn transform a struct with the connection data to a connection string
```

Comment:
Add a dot at the end of the line?

Reply:
Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034